src/styles/card.scss

```diff
@@ -0,0 +1,96 @@
+@import "variables";
+
+.card {
+  width: 167px;
```

Comment:
I believe that the width of the card should be 200px, doesn't it? add box-sizing border box so paddings will be counted in that value of 200px